test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -233,7 +260,11 @@ export default class PerfCompare extends Component {
 
         let baselineRunJSON = new JenkinsRunJSON(resBenchmarkBaselineJson);
```

Comment:
I think we need a better name than `JenkinsRunJSON` since it doesn't really tell what it does. Maybe we can call it `ExtractRelevantJenkinsTestResults`.